### PR TITLE
Generate requirements from minimal

### LINKED ISSUE #13: Pin requirements
https://github.com/edmundmok/mealpy/blob/2b7e67405083b37e62795a5eff6374de5046eede/requirements.txt#L1-L2
These requirements should be pinned so that install is the same. Set up [Yelp/requirements-tools](https://github.com/Yelp/requirements-tools) to do this.

### PR DESCRIPTION
**Pull request by ipwnponies**
The only addition is `requirements-tools` to `requirements-dev-minimal.txt`. The rest is autogenerated.

This fixes #13 so that users will install this with same requirements as what was developed on.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,25 @@
...
+
+```bash
+make venv
+```
+
 ## Help
 
 ```bash
-python mealpy.py --help
+venv/bin/python mealpy/mealpy.py --help
```
Comment (ipwnponies) [inline README.md]: Not sure if this needs to be stated. But I guess some users may not be python users and not necessarily aware of need to activate virtualenv.

Comment (edmundmok) [reply README.md]: Maybe just add the activation step `source venv/bin/activate` above? So that we can simply use `python` in all the command examples.

**Review by edmundmok — APPROVED** [review]
How are the requirement versions generated? Is there a command to use?

**Comment by ipwnponies** [comment]
This uses [requirements-tools](https://github.com/Yelp/requirements-tools#upgrade-requirements). The documentation is quite a good read.

File: README.md (hunk 1 above)
Comment (ipwnponies) [reply README.md]: I updated instructions.

**Pull request merged**